openfermionpyscf/__init__.py

```diff
@@ -17,6 +17,8 @@
 from ._version import __version__
```

Comment:
while you're at it, this should also be imported last (alphabetical ordering). I realize this was here before you changed anything.